### PR TITLE
Fix drone

### PR DESCRIPTION
**Pull request by MorrisJobke**

### CODE REVIEW

**Comment by MorrisJobke** [comment]
@danxuliu @icewind1991 I needed to disable the acceptance and object storage tests temporarily because they rely on the fact, that the service (selenium and fake-s3) on localhost, but actually with drone 0.7 this is not the case anymore (because they matched with the docker-compose behaviour). Now we need to use the hostnames `selenium` and `fake-s3` to connect to those services and I was not quickly able to do this change. But to let not fail all the other PRs I would go for merging this and then bring back those CI jobs soonish with separate PRs.

File: autotest.sh [hunk 1]
```diff
@@ -46,6 +46,8 @@ function print_syntax {
 	echo -e "\nIf no arguments are specified, all tests will be run with all database configs" >&2
 }
 
+pwd
```
Comment (icewind1991) [inline autotest.sh]: is this change intended?

Comment (MorrisJobke) [inline autotest.sh]: no :P

**Review by icewind1991 — APPROVED** [review]
drone seems happy 👍
